tests/unit/framework/test_crawlers.py

```diff
@@ -204,3 +210,39 @@ def test_runtime_backend_save_table(mocker):
             "first STRING NOT NULL, second BOOLEAN NOT NULL",
         )
         rb._spark.createDataFrame().write.saveAsTable.assert_called_with("a.b.c", mode="append")
+
+
+def test_runtime_backend_save_table_with_row_containing_none(mocker):
+    from unittest import mock
+
+    with mock.patch.dict(os.environ, {"DATABRICKS_RUNTIME_VERSION": "14.0"}):
+        pyspark_sql_session = mocker.Mock()
+        sys.modules["pyspark.sql.session"] = pyspark_sql_session
+
+        rb = RuntimeBackend()
+
+        rb.save_table("a.b.c", [Foo("aaa", True), Foo("bbb", False), Foo("bbb", None)])
```

Comment:
```suggestion
        rb.save_table("a.b.c", [Foo("aaa", True), Foo("bbb", False), Foo("ccc", None)])
```
so that it's more clear from reading the test